tox_docker.py

```diff
@@ -110,6 +110,8 @@ def getint(reader, key):
             })
         if reader.getstring("ports"):
             image_configs[image]["ports"] = reader.getlist("ports")
+        if reader.getstring("links"):
+            image_configs[image]["links"] = [link.strip() for link in reader.getstring("links").split(' ') if link.strip()]
```

Comment:
can we use `reader.getlist()` (I think that's what it's called) -- that lets you specify it in the typical tox way